Simplenote/src/main/res/values/dimens.xml

```diff
@@ -56,6 +56,7 @@
     <dimen name="sort_switch_width">40dp</dimen>
     <dimen name="style_stroke">1dp</dimen>
     <dimen name="style_stroke_activated">2dp</dimen>
+    <dimen name="style_width">-1px</dimen>
```

Comment:
I'm wondering if this could benefit from a comment stating this is faking `match_parent`?

Reply:
Good idea.  I added the comment in e41b0b8d.